web/profiles/custom/yalesites_profile/themes/custom/ys_admin_theme/ys_admin_theme.info.yml

```diff
@@ -0,0 +1,36 @@
+name: YaleSites Admin Theme
+type: theme
+base theme: gin
+description: "Admin theme for YaleSites."
+core_version_requirement: ^8.9 || ^9 || ^10
+
+libraries:
+  - claro/global-styling
+  - gin/gin
+
+# Regions
+regions:
+  header: "Header"
+  pre_content: "Pre-content"
+  breadcrumb: Breadcrumb
+  highlighted: Highlighted
+  help: Help
+  content: Content
+  page_top: "Page top"
+  page_bottom: "Page bottom"
+  sidebar_first: "First sidebar"
+regions_hidden:
+  - sidebar_first
+
+components:
```

Comment:
Is this accidental copy paste?

Reply:
No, the admin theme needs to know about these components because of the way layout builder works... I'll verify while/after I get some things ironed out in the authoring experience stuff, but if we use layout builder on the backend (which I think we will) the admin theme needs to know where our layouts are... which might actually end up pointing to your module instead of this theme in the end...